BazelExtensions/workspace.bzl

```diff
@@ -184,8 +198,8 @@ pod_repo_ = repository_rule(
         "install_script_tpl": attr.string(),
         "inhibit_warnings": attr.bool(default=False, mandatory=True),
         "trace": attr.bool(default=False, mandatory=True),
-        "enable_modules": attr.bool(default=True, mandatory=True),
-        "generate_module_map": attr.bool(default=True, mandatory=True),
+        "enable_modules": attr.bool(default=False, mandatory=True),
```

Comment:
I agree that we should turn these off by default, it's an internal convention that we default them to on, and it's probably annoying since they don't work so well. We should update the `update_pods.py` / Vendorizing script to have these defaults as well - that could be a followup